src/content/app/tools/blast/components/blast-settings/BlastSettings.tsx

```diff
@@ -35,7 +37,6 @@ import {
   getBlastJobName
 } from 'src/content/app/tools/blast/state/blast-form/blastFormSelectors';
 import {
-  setBlastDatabase,
   setBlastProgram,
```

Comment:
This might not be related to this change but more of a general question for blast, should the setBlastProgram also be moved to the blast hook? How do we know which one to move to the hook, its just a worry about there will be a mixture between hooks and redux function.

Reply:
so after looking at the hook useBlastForm, it feels like all of them has moved to the hooks and we should use them instead, is that correct?

Reply:
Added a couple of more functions to useBlastForm. There are still several places where we are updating blastFormSlice by dispatching actions directly from inside components; but trying to wrap those actions inside the useBlastForm hook felt unnecessary and clumsy.